### PR TITLE
#161243659 Add the application and file structure

### PR DESCRIPTION
**Pull request by KMaina**
#### What does this PR do?
Makes the application structure and folder hierarchy to be used by all features
#### Description of Task to be completed?
Inspect the application structure and folder hieracrchy for further review
#### How should this be manually tested?

- Clone the repo locally

- cd into the store-manager folder

- run `git pull https://github.com/KMaina/Store-Manager-API.git ch-add-file-structure-161243659`

- review the application structure

#### Any background context you want to provide?
This is generally how the app will be organized but some files may be added on later
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/161243659

### CODE REVIEW

**Comment by EdwardMudaida** [comment]
Use imperative, present tense: “change” not “changed” nor “changes” for your PR title.

File: instance/config.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+class ProductionConfig(Config):
+    """Configurations for Production."""
+    pass
+
+app_config = {
+    'development': DevelopmentConfig,
+    'testing': TestingConfig,
+    'staging': StagingConfig,
+    'production': ProductionConfig,
+}
```
Comment (ghost) [inline instance/config.py]: Add a new line here.

File: instance/config.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"""The configuration for the app"""
+
+class Config(object):
+    """Parent configuration class."""
+    pass
```
Comment (ghost) [inline instance/config.py:5]: Why do you have `pass` here in place of the environment configuration?

Comment (KMaina) [reply instance/config.py:5]: I was planning on using the contents of this branch as a template such that any feature branch would branch off this one. From there I would use the actual environment variables

**Pull request merged**